app/main.py

```diff
@@ -1 +1,42 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is defined as a class attribute. This means it is shared across all instances of the Animal class. If this is intended to track all living animals globally, it's fine. Otherwise, consider using an instance attribute if you want each animal to have its own list.